src/empty.js

```diff
@@ -1,21 +1,36 @@
 var _curry1 = require('./internal/_curry1');
-var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` returns an empty list for any argument, except when the argument satisfies the
- * Fantasy-land Monoid spec. In that case, this function will return the result of invoking
- * `empty` on that Monoid.
+ * Returns the empty value of its argument's type. The argument must be a
+ * [Monoid](https://github.com/fantasyland/fantasy-land#monoid), Array,
+ * Object, or String.
```

Comment:
Fair point. It was mainly suggested because it is modifying the existing behaviour of `empty` which currently defaults to an empty array and would be good to have some docs to refer people back to if questions are raised.